config.nix

```diff
@@ -258,17 +258,22 @@ let
         # this turns members like "foo/*" into [ "foo/bar" "foo/baz" ]
         # as in https://github.com/rust-analyzer/rust-analyzer/blob/b2ed130ffd9c79de26249a1dfb2a8312d6af12b3/Cargo.toml#L2
         expandMember = member:
-          if lib.hasSuffix "/*" member
+          if lib.hasInfix "*" member
```

Comment:
what about `hasSuffix "/*" || hasSuffix "/*/"`? This is a tad misleading, I first assumed it handled cases like `foo/*/bar` expanding to `foo/quux/bar`.

Reply:
Oh, good catch, that makes sense.